reconcile/test/test_openshift_namespaces.py

```diff
@@ -116,3 +121,17 @@ def testDeleteAbsentNamespace(self):
         oc = self.oc_clients[c1]
         oc.delete_project.assert_not_called()
         oc.new_project.assert_not_called()
+
+    def testErrorHandlingProjectExists(self):
```

Comment:
No camelCase, for methods, please:
```suggestion
    def test_error_handling_project_exists(self):
```

Reply:
Fixed in all cases.